Week1/hw3/trip.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="trip.css" />
+    <title>두근두근 여행</title>
+</head>
+<body>
+    <section>
+        <header>
+            <h1 class="title">💜 두근두근 유럽 여행</h1>
+        </header>
+        <div class="preview">
+            <ul>
+                <li >
+                    <img src="./images/london3.jpg" alt="아이스크림"/>
+                </li>
```

Comment:
이 header는 문서 전체에서의 header를 나타내주니까,
```suggestion
        <header>
            <h1 class="title">💜 두근두근 유럽 여행</h1>
        </header>
        <main>
            <section class="preview">
                <ul>
                    <li >
                        <img src="./images/london3.jpg" alt="아이스크림"/>
                    </li>
```
이런식의 구조가 좀 더 적합할 것 같다는 의견입니다!

Reply:
main을 사용해서 웹 문서의 구조를 더 확실하게 나타낼 수 있군요..! 감사합니다!!👍